src/index.html

```diff
@@ -12,7 +12,50 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="stars__body">
+    <div class="wrapper">
+      <div class="stars stars--0">
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+      </div>
+      <div class="stars stars--1">
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+      </div>
+      <div class="stars stars--2">
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+      </div>
+      <div class="stars stars--3">
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+      </div>
+      <div class="stars stars--4">
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+      </div>
+      <div class="stars stars--5">
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+      </div>
+    </div>
```

Comment:
The class names 'stars__body', 'stars_star', and the modifiers 'stars--0', 'stars--1', etc., do not represent the meaning of the content but rather the styles or tag names. According to the checklist, class names should represent the meaning of the content, not the styles or tag names.